doc/changelog.rst

```diff
@@ -1,28 +1,12 @@
 Changelog
 =========
 
-Development branch
-------------------
-
-Work in progress
-
-Description
-
-- n / a
-
-
-Detailed Notes
-
-- n / a
-
-.. _PRXXX: https://github.com/CrayLabs/SmartDashboard/pull/XXX
-
-
 0.0.2
 -----
 
-Released on *TBD*, 2023
+Released on 14 December, 2023
 
 Description
 
-- Initial release of SmartDashboard
\ No newline at end of file
+- The initial release of SmartDashboard includes capabilities for viewing 
+experiment entity properties and statuses.
```

Comment:
Small pedantic suggestion for future readability
```suggestion
- The initial release of SmartDashboard includes capabilities for viewing 
  experiment entity properties and statuses.
```

Reply:
fixed!